### PR TITLE
[F30] Prepare for fips-mode-setup being split out of crypto-policies

### PR DESCRIPTION
**Pull request by keszybz**
Same as https://github.com/rhinstaller/anaconda/pull/1892.

### CODE REVIEW

**Comment by jkonecny12** [comment]
Jenkins, it's ok to test.

**Comment by jkonecny12** [comment]
You have to solve this pylint issue before merging:
```
E1101(no-member):334,11: doInstall.run_pre_install: Module 'pyanaconda.flags' has no 'cmdline' member
```

File: pyanaconda/installation.py [hunk 1]
```diff
@@ -331,6 +331,9 @@ def run_pre_install():
 
         if can_install_bootloader:
             payload.requirements.add_packages(storage.bootloader.packages, reason="bootloader")
+        if flags.cmdline.get("fips") == "1":
```
Comment (jkonecny12) [inline pyanaconda/installation.py]: To make this work you have to change the code to this.
```suggestion
        if flags.flags.cmdline.getbool("fips"):
```

Sorry for the ``flags.flags`` that is just a wrong import but we will fix that in the future later.

**Comment by keszybz** [comment]
Thanks for the suggestion. Updated.

**Pull request merged**